docs/frequently-asked-questions.md

```diff
@@ -108,3 +108,22 @@ Documentation on how to do that can be found [here](https://giscience.github.io/
 
 If CH have not been prepared, matrix calculation will fall back to the Dijkstra
 algorithm, which is way slower and responsible for the slow response.
+
+## When does my quota reset?
+When you check our [plans](https://openrouteservice.org/plans/), you'll see
+that our endpoints have a daily and a minutely request limit.
+
+The daily limit is reset after 24h, starting from the first time you request
+anything. Thus, your 24h-window might shift over the days.
+
+As an example, let's look at the `directions` endpoint and its default limit of
+2000 requests per day.  If you request 1000 directions at 3pm today and try to
+request 1500 tomorrow at 10am, the last 500 of those will fail, since your
+quota only resets at 3pm tomorrow. After 3pm, you can issue up to 2000 requests
+again, but if you do so at 6pm, you'll have to wait until 6pm the day after
+tomorrow until you can request any more directions.
+
+The minutely limit is enforced as a sliding window, meaning that any
+consecutive period of 60 seconds may only contain 40 directions requests.
+
+If you run into either the daily or the minutely limit, you will receive a `429 - Too many requests` HTTP error.
```

Comment:
if you use up your daily limit you run into 403 not allowed (yay tyk)
See also response of collaborative plan:
```
stay within your request limits by checking for 429 (minutely limit) and 403 (daily limit) HTTP responses
```

also see https://ask.openrouteservice.org/t/what-is-the-logic-behind-x-ratelimit-reset/3040
and the linked topic which might be relevant here

Reply:
We could also just link to the tyk explanation for this, but i think the description above is better.
Please compare and decide:
https://tyk.io/docs/basic-config-and-security/control-limit-traffic/request-quotas/

Reply:
Even with the knowledge I have, I find it hard to grasp the tyk explanation - it seems more tyk-admin-focused than user focused.

The other comment has been addressed.